src/nupic/os/Timer.cpp

```diff
@@ -66,38 +66,6 @@ static UInt64 getCurrentTime()
   return (UInt64)(v.QuadPart) - initialTicks_;
 }
 
-#elif defined(NTA_PLATFORM_darwin)
```

Comment:
I think there was a reason that darwin was targeted here. We had a bunch of destructive build changes so I am not surprised that the proper variables stopped getting set. And I am curious why there is a bug in the code now. What is the bug that you see?